object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,32 @@
+/* exported defaults */
+
+/*
+
+***************************** PSEUDOCODE *****************************
+
+- create a counter
+- look through each property of the source and compare it to the target
+- if there is a match, add one to the counter
+- if the counter is equivalent to 0, add the property to the target
+- reset the counter
+- return the target object
+
+**********************************************************************
+
+*/
+
+function defaults(target, source) {
+  var counter = 0;
+  for (var key in source) {
+    for (var innerKey in target) {
+      if (key === innerKey) {
+        counter++;
```

Comment:
Instead of using a counter here where you are essentially checking if the value is zero or not zero, this seems like a perfect opportunity for a variable holding a boolean value.